### PR TITLE
Fix the issue of change clustername page crash and change content of …

### PR DESCRIPTION
**Pull request by LeoHongyi**
…constant of warning

### CODE REVIEW

File: src/dashboard/src/pages/ClusterStatus/components/PhysicalClusterNodeStatus.tsx [hunk 1]
```diff
@@ -60,31 +62,50 @@ export const PhysicalClusterNodeStatus = (props: PhClusterNSType) => {
...
                           return (
-                            <Typography variant="subtitle2" component="b" gutterBottom>
-                              {`[${pod}]`}
-                            </Typography>
+                            <>
+                              <Typography variant="subtitle2" component="b" gutterBottom>
+                                {`[${pod}]`}
+                              </Typography>
+                              <br/>
+                            </>
```
Comment (Gerhut) [inline src/dashboard/src/pages/ClusterStatus/components/PhysicalClusterNodeStatus.tsx]: Use `<p>` with `<strong>`

File: src/dashboard/src/pages/ClusterStatus/index.tsx [hunk 2]
```diff
@@ -92,6 +93,11 @@ const ClusterStatus: FC = () => {
       });
       const filterclusters = convertToArrayByKey(clusters, 'id');
       setSelectedValue(filterclusters[0]);
+      if (localStorage.getItem("selectedCluster")) {
+        setSelectedValue((String)(localStorage.getItem("selectedCluster")));
```
Comment (Gerhut) [inline src/dashboard/src/pages/ClusterStatus/index.tsx]:
```suggestion
        setSelectedValue(String(localStorage.getItem("selectedCluster")));
```

File: src/dashboard/src/pages/Jobs/index.tsx [hunk 3]
```diff
@@ -365,527 +377,528 @@ const Jobs: React.FC = (props: any) => {
...
+                /> : null}
+                {/*/!*</Container>*!/*/}
+              </DLTSTabPanel> : <CircularProgress/>
         }
         <DLTSSnackbar message={message}
-          open = {openKillWarn || openApproveWarn || openPauseWarn || openResumeWarn || openUpatePriorityWarn}
-          handleWarnClose={()=>{handleWarnClose()}}
-          autoHideDuration={1000}
+                      open = {openKillWarn || openApproveWarn || openPauseWarn || openResumeWarn || openUpatePriorityWarn}
+                      handleWarnClose={()=>{handleWarnClose()}}
```
Comment (Gerhut) [inline src/dashboard/src/pages/Jobs/index.tsx]: Why additional func call here?

**Comment by hongzhili** [comment]
@LeoHongyi , Please address Qixiang's comments.

**Pull request merged**